book/validity/experimental/celestia.md

```diff
@@ -0,0 +1,32 @@
+# Celestia Data Availability
+
+This section only describes the additional requirements to use Celestia DA. Most of the requirements are the same as the `op-succinct` service. Please refer to the [Proposer](../proposer.md) section for the basic setup.
+
+## Environment Setup
+
+To use Celestia DA, you need additional environment variables in the `.env` file:
+
+| Parameter | Description |
+|-----------|-------------|
+| `CELESTIA_CONNECTION` | URL of the Celestia RPC endpoint. |
+| `NAMESPACE` | Namespace ID for the Celestia DA. |
```

Comment:
Can you use the definition of a namespace from the Celestia docs and link to it?